com.mimacom.ddd.system.tests/src/com/mimacom/ddd/system/tests/ParsingTest.xtend

```diff
@@ -48,51 +49,64 @@ class ParsingTest {
 
   @Test
   def void parseBaseTypes() {
-    val content = dimParseHelper.parse(new URL(BASE_TYPES_URI.toString).openStream, BASE_TYPES_URI, null, resourceSet);
+    val content = dimParseHelper.parse(new URL(BASE_TYPES_URI.toString).openStream, BASE_TYPES_URI, null, resourceSet)
     assertNotNull(content)
+    assertNoErrorsOnResource(content.eResource)
   }
 
   @Test
   def void parseSystemFunctions() {
     val injector = new DmxInjectorProvider().injector
     val dmxParseHelper = injector.getInstance(ParseHelper)
     val content = dmxParseHelper.parse(new URL(SYSTEM_FUNCTIONS_URI.toString).openStream, SYSTEM_FUNCTIONS_URI, null,
-      resourceSet);
+      resourceSet)
     assertNotNull(content)
+    assertNoErrorsOnResource(content.eResource)
   }
 
   @Test
   def void parseAssignments() {
     val injector = new DmxInjectorProvider().injector
     val dmxParseHelper = injector.getInstance(ParseHelper)
     val content = dmxParseHelper.parse(new URL(ASSIGNMENTS_URI.toString).openStream, ASSIGNMENTS_URI, null,
-      resourceSet);
+      resourceSet)
     assertNotNull(content)
+    assertNoErrorsOnResource(content.eResource)
   }
 
   @Test
   def void parseCustomTypes() {
     val content = dimParseHelper.parse(new URL(CUSTOM_TYPES_URI.toString).openStream, CUSTOM_TYPES_URI, null,
-      resourceSet);
+      resourceSet)
     assertNotNull(content)
+    assertNoErrorsOnResource(content.eResource)
   }
 
   @Test
   def void parseMathFunctions() {
     val injector = new DmxInjectorProvider().injector
     val dmxParseHelper = injector.getInstance(ParseHelper)
     val content = dmxParseHelper.parse(new URL(MATH_FUNCTIONS_URI.toString).openStream, MATH_FUNCTIONS_URI, null,
-      resourceSet);
+      resourceSet)
     assertNotNull(content)
+    assertNoErrorsOnResource(content.eResource)
   }
 
   @Test
   def void parseSystemTypes() {
     val injector = new DmxInjectorProvider().injector
     val dmxParseHelper = injector.getInstance(ParseHelper)
     val content = dmxParseHelper.parse(new URL(SYSTEM_TYPES_URI.toString).openStream, SYSTEM_TYPES_URI, null,
-      resourceSet);
+      resourceSet)
     assertNotNull(content)
+    assertNoErrorsOnResource(content.eResource)
+  }
+  
+  def void assertNoErrorsOnResource(Resource resource) {
```

Comment:
@oliver-reinhard : please find an updated version of the system test assertions here.

Reply:
Nice, thanks.